### DISCUSSION TITLE
Some insights about Mobile UI Designing

### DISCUSSION THREAD

**Discussion by sithijasanthiya** [pawan-live/timetable-app#124] [General]
![dp and sp 1](https://github.com/pawan-live/timetable-app/assets/110617750/5ed4605e-8f9e-4de0-891a-866ba0ae9402)

![dp and sp 2](https://github.com/pawan-live/timetable-app/assets/110617750/fc8f8e70-8c9e-45ce-a361-1c3a1b5077f7)

**Comment by pawan-live** [comment]
Nice!

**Comment by sithijasanthiya** [comment]
Reference :  https://www.altova.com/manual/MobileTogether/mobiletogetherdesigner/mtdobjsfeatures_sizes.html#:~:text=of%20differing%20resolutions.-,Scale%2Dindependent%20pixels%20(sp),in%20the%20device's%20system%20settings.
